gui/src/app/Scripting/Runtime/load_args.py

```diff
@@ -0,0 +1,27 @@
+def rename_sampling_options(k):
+    """
+    convert between names used in
+    Stan-Playground and CmdStanPy
+    """
+
+    if k == "init_radius":
+        return "inits"
+    if k == "num_warmup":
+        return "iter_warmup"
+    if k == "num_samples":
+        return "iter_sampling"
+    if k == "num_chains":
+        return "chains"
+    if k == "seed":
+        return "seed"
+
+    raise ValueError(f"Unknown sampling option: {k}")
+
+
+if os.path.isfile(os.path.join(HERE, "sampling_opts.json")):
```

Comment:
Where is HERE defined?

Reply:
Never mind, I see, this is meant to be part of a script that's getting assembled later.

Reply:
Particularly through Github :)